src/DependencyInjection/EnumTranslationCompilerPass.php

```diff
@@ -11,21 +11,14 @@
 class EnumTranslationCompilerPass implements CompilerPassInterface
 {
     /**
-     * @see \Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface::process()
+     * {@inheritdoc}
      */
     public function process(ContainerBuilder $container)
     {
-        foreach ($container->getParameter('kernel.bundles') as $name => $class) {
-            // Special case when using the EntityBundle
-            $base_dir = ($class === 'Hostnet\Bundle\EntityBundle\HostnetEntityBundle')
-                ? $container->getParameter(sprintf('hostnet.entity.%s.path', $container->underscore($name)))
-                : dirname((new \ReflectionClass($class))->getFilename());
-
-            if (!is_dir($dir = $base_dir . '/Resources/translations')) {
-                continue;
-            }
-
-            $this->registerEnums($container, glob($dir . "/enum.*.*"));
+        foreach ($container->getDefinition('translator.default')->getArgument(3)['resource_files'] as $files) {
+            $this->registerEnums($container, array_filter($files, function ($file) {
+                return preg_match('~/enum\.[a-z]+\.[a-z]+$~', $file) === 1;
```

Comment:
This is tested via https://github.com/hostnet/entity-translation-bundle/pull/8/files#diff-1cf5deb72b2e782d9c072aa5c4d731beR14 via symfony files e.g. security.az.xlf.

This is also tested by the exception test: https://github.com/hostnet/entity-translation-bundle/pull/8/files#diff-c63bc4e743e5eccaa0948c7a98764e14R35